src/MessageContainer.js

```diff
@@ -107,7 +107,7 @@ export default class MessageContainer extends React.PureComponent {
           initialNumToRender={MAX_RENDER_ITEM}
           maxToRenderPerBatch={MAX_RENDER_ITEM}
           windowSize={MAX_RENDER_ITEM}
-          removeClippedSubviews
+          removeClippedSubviews={false}
```

Comment:
Did you face any problem? Using false has performance implications.

Reply:
The doc mention bug related to this props so I try to see that change but no https://facebook.github.io/react-native/docs/flatlist.html#removeclippedsubviews